src/resolvers/projectResolver.test.ts

```diff
@@ -1830,6 +1833,81 @@ function likedProjectsByUserIdTestCases() {
   });
 }
 
+function projectByIdTestCases() {
+  it('should return project with id', async () => {
+    const project = await saveProjectDirectlyToDb(createProjectData());
+    const result = await axios.post(graphqlUrl, {
+      query: projectByIdQuery,
+      variables: {
+        id: project.id,
+      },
+    });
+    assert.equal(result.data.data.projectById.id, project.id);
+    assert.equal(result.data.data.projectById.slug, project.slug);
+  });
+
+  it('should return project null with invalid id', async () => {
+    const result = await axios.post(graphqlUrl, {
+      query: projectByIdQuery,
+      variables: {
+        // To make use id is invalid
+        id: 9999999,
+      },
+    });
+    assert.equal(
+      result.data.errors[0].message,
+      'Cannot return null for non-nullable field Query.projectById.',
+    );
+  });
+
+  it('should return reaction when user liked the project', async () => {
+    const project = await saveProjectDirectlyToDb(createProjectData());
+    const user = await saveUserDirectlyToDb(generateRandomEtheriumAddress());
+    const reaction = await Reaction.create({
+      userId: user.id,
+      projectId: project.id,
+    }).save();
+    const result = await axios.post(graphqlUrl, {
+      query: projectByIdQuery,
+      variables: {
+        id: project.id,
+        connectedWalletUserId: user.id,
+      },
+    });
+    assert.equal(result.data.data.projectById.id, project.id);
+    assert.equal(result.data.data.projectById.reaction.id, reaction.id);
+  });
```

Comment:
@aminlatifi  I wrote some test cases for `projectById` query and this test fails, I don't know it's my test case problem or your code

error message
```
       should return reaction when user liked the project:
     QueryFailedError: null value in column "reaction" of relation "reaction" violates not-null constraint
```

Reply:
Thanks for making it complete by a test. The reaction creation on test had an issue not the graphql endpoint. It's fixed by this
https://github.com/Giveth/impact-graph/pull/358/commits/e979c95e2c3a07a1524abdb22833e5c7dee2c98d

Reply:
Great, thank you man